ONSdigital/dp-frontend-search-controller
# allow 3rd level subtopics

### What

Refactor aggregated data subtopic pages to allow multi level subtopics
[Trello Card](https://trello.com/c/9HgeOu40/2177-refactor-aggregated-data-subtopic-pages-to-allow-3rd-level-subtopics)

### How to review

- Check code is ok
- Pull branch
- set EnableReworkedDataAggregationPages to 'true'
- run application and try accessing a route with topic, subtopic and subsubtopic in its url from your browser
- Ensure correct data is displayed and cache working.

### Who can review

Anyone
